homeassistant/components/template/strings.json

```
@@ -202,9 +203,39 @@
           }
         },
         "title": "Template switch"
+      },
+      "weather": {
+        "data": {
+          "device_id": "[%key:common::config_flow::data::device%]",
+          "name": "[%key:common::config_flow::data::name%]",
+          "condition": "Condition",
+          "humidity": "Humidity",
+          "temperature": "Temperature",
+          "temperature_unit": "Temperature unit",
+          "forecast_daily": "Forecast daily",
+          "forecast_hourly": "Forecast hourly"
+        },
+        "data_description": {
+          "device_id": "[%key:component::template::common::device_id_description%]"
+        },
+        "sections": {
+          "advanced_options": {
+            "name": "[%key:component::template::common::advanced_options%]",
+            "data": {
+              "availability": "[%key:component::template::common::availability%]"
+            }
+          }
+        },
+        "title": "Template weather"
       }
     }
   },
+  "issues": {
+    "deprecated_weather_options": {
+      "title": "Deprecation: Legacy options in {entity_name}",
+      "description": "The legacy options for template weather are being removed in 2026.1.\n\nPlease update the following YAML options for {entity_id} ({entity_name}): \n\n{changes}"
```

Same small change as in previous PR for the `template` integration for consistent translations.
And an excessive space to remove along the way:

```suggestion
      "title": "Deprecated legacy options in {entity_name}",
      "description": "The legacy options for template weather are being removed in 2026.1.\n\nPlease update the following YAML options for {entity_id} ({entity_name}):\n\n{changes}"
```